server/server.go

```diff
@@ -682,3 +729,18 @@ func (s *Server) getConnector(id string) (Connector, error) {
 
 	return conn, nil
 }
+
+type logRequestKey string
+
+const (
+	LogRequestKeyRequestID logRequestKey = "request_id"
+	LogRequestKeyRemoteIP  logRequestKey = "client_remote_addr"
```

Comment:
```suggestion
	RequestKeyRequestID logRequestKey = "request_id"
	RequestKeyRemoteIP  logRequestKey = "client_remote_addr"
```

Drop the log part: we don't use it for anything else, but at the point of insertion it doesn't really matter IMO.